plasmapy/analysis/fit_functions.py

```diff
@@ -221,6 +216,15 @@ def curve_fit_results(self):
         """
         return self._curve_fit_results
 
+    @property
+    def FitParamTuple(self):
```

Comment:
[optional] Could add a type hint annotation if you'd like.  Not necessary though.

Reply:
Wouldn't even know how to properly do this because `FitParamTuple` is the type/class.